app/views/machines/show.html.erb

```diff
@@ -1 +1,12 @@
 <h1><%= @machine.location %> Vending Machine</h1>
+<%= @machine.snacks.each do |snack| %>
+
+  <section id="snack-<%= snack.id %>">
+    <%= snack.name %>
+    <%= '%.2f' % snack.price.round(2) %>
```

Comment:
I like your creativity here... you can take a look at an action view helper `number_to_currency` to help with this formatting.